release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1217,10 +1217,22 @@ OpenShift SDN network plugin was deprecated in 4.15 and 4.16. With this release,
 [id="ocp-4-17-cloud-cred-operator-bug-fixes_{context}"]
 ==== Cloud Credential Operator
 
+* Previously, secrets in the cluster were fetched in a single call.
+When there were a large number of secrets, this caused the API to time out.
+With this release, the Cloud Credential Operator fetches secrets in batches limited to 100 secrets. This change prevents timeouts when there are large number of secrets in the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-41233[*OCPBUGS-41233*])
+
+* Previously, the Cloud Credential Operator reported an error when the `awsSTSIAMRoleARN` role was not present on a cluster that uses manual mode with AWS STS. With this release, the Cloud Credential Operator is updated to no longer report this as an error. (link:https://issues.redhat.com/browse/OCPBUGS-33566[*OCPBUGS-33566*])
```

Comment:
```suggestion
* Previously, the Cloud Credential Operator reported an error when the `awsSTSIAMRoleARN` role was not present on a cluster that used manual mode with AWS Security Token Service. With this release, the Cloud Credential Operator no longer reports this as an error. (link:https://issues.redhat.com/browse/OCPBUGS-33566[*OCPBUGS-33566*])
```
See [AWS Security Token Service ](https://docs.aws.amazon.com/STS/latest/APIReference/welcome.html) for nomenclature.